source/integrations.txt

```diff
@@ -32,41 +32,36 @@ This page describes the following third-party integrations:
 .. TODO (add after JAVA-3580 is finished):
    - :ref:`graalvm-integration`
 
-.. note::
+.. important::
 
-   MongoDB does not own the tools included on this page.
+   The integrations on this page are not official MongoDB partners.
 
 .. _spring-framework:
 
 Spring Data
 -----------
 
-The `Spring <https://spring.io/>`__ Java framework offers flexible third-party libraries,
-streamlined code features, and performance improvements to facilitate application
-development.
+`Spring <https://spring.io/>`__ is a Java framework that provides infrastructure support
+for Java applications. Spring's modules enable aspect-oriented programming, authentication
+protocols, inversion of control, and other services to facilitate application development.
 
-Spring Data, a Spring umbrella project, focuses on database access for 
-Spring-powered applications. Several Spring Data features simplify data access, including
-a high-level interface for database interaction and database query generation.
+Spring offers the Spring Data sub-framework for database access. Spring Data's features include
+map-reduce frameworks, cloud-based data services, and an access layer for relational and
+non-relational databases. To learn more about the Spring Data project, see `Spring Data
+<https://spring.io/projects/spring-data>`__.
 
 Using Spring Data and MongoDB
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-`Spring Data MongoDB <https://spring.io/projects/spring-data-mongodb/>`__ is one of the
-modules included under the Spring Data umbrella. This module supports the use of MongoDB
-databases during Spring application development. Additionally, Spring Data MongoDB reduces
-the boilerplate code and MongoDB API knowledge required for MongoDB database access. The
-module offers the following features:
-
-- ``MongoTemplate`` helper class that implements the ``MongoOperation`` interface.
-  This interface provides methods such as ``find()``, ``insert()``, and ``update()``
-  to perform MongoDB operations.
-- Exception translation that allows for custom, portable exception handling code
-  without losing error information provided by the {+driver-long+}.
-- Converter, called ``MappingMongoConverter``, to map domain objects to MongoDB documents.
-  You can also write a custom converter.
-- ``GridFsOperations`` and ``ReactiveGridFsOperations`` interfaces for accessing
-  the GridFS filesystem.
+`Spring Data MongoDB <https://spring.io/projects/spring-data-mongodb/>`__ allows you to access
+MongoDB databases by using a Spring-based programming model. For instructions on using
+Spring Data MongoDB to build an application and interact with MongoDB, see the 
+`Accessing Data with MongoDB <https://spring.io/guides/gs/accessing-data-mongodb>`__ guide in
+the Spring documentation.
```

Comment:
Suggestion:
Related to [this comment](https://github.com/mongodb/docs-java/pull/511/#discussion_r1486783015), I think the placement of "how to use this" ("For instructions...") between the "what it is" and "what are the features" parts of the information can interrupt the flow.

I think ordering the info the following way could flow better:

"What it is"
"What are the features"
"How to use this"

Applies to all the sections.